### ISSUE TITLE
[Feature Request] Live Photo iOS Support

### ISSUE THREAD

**Issue by lingfeishengtian** [FilenCloudDienste/filen-mobile#173]
This feature has been requested a few times, but is challenging to implement. I feel like a good way to implement Live Photos is to upload the MOV and HEIC of the file (as that's all a Live Photo is). However, viewing a Live Photo yields a bigger challenge as this would require searching the directory for an exact name MOV which may or may not be the Live Photo pair (would need input from user). In addition, saving the Live Photo pair back to the gallery would also prove a challenge. My proposal is to scan for a pair and **suggest the user** to combine them if found (user is able to refuse combination). 

However, another way Live Photos could be implemented is if a folder with, for example, an arbitrary extension such as .live is added with the MOV and HEIC inside.  However, rendering the Live Photo on Android would prove to be a challenge.

/src/lib/services/cameraUpload)/cameraUpload.ts => getFile()
Check asset mediaSubtypes =>  if "livePhoto" exists, then return two files to be uploaded where

(there's probably a better way to do this)
```js
assetInfo.filename.substring(0, assetInfo.filename.lastIndexOf('.')) + ".MOV" AND "HEIC"
```

**Comment by Dwynr** [comment]
Hi!

Thank you for your input. I agree that we should implement this feature, though I'd suggest doing it a bit different:

- Add a settings toggle to enable/disable live photo to MOV conversion
- If enabled, convert live photo to MOV and upload, if not, keep it in HEIC/JPG

With the latest additions to the app video thumbnails are a thing, so the live photos would be displayed as a video in the photos screen (same for Android as long as the underlying phone supports creating thumbnails from MOV files).

**Comment by lingfeishengtian** [comment]
I agree with the switch approach, however, the MOV is a lower quality than the image so doing it this way will lose some data.

**Comment by lingfeishengtian** [comment]
Being implemented with #176
